internal/controller/kyma_controller.go

```diff
@@ -102,12 +102,11 @@ func (r *KymaReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.
 	kyma := &v1beta2.Kyma{}
 	if err := r.Get(ctx, req.NamespacedName, kyma); err != nil {
 		if !util.IsNotFound(err) {
```

Comment:
Great you catched the bug here. It would be good to also improve this conditional. I consider double negation as hard to read: "if not is not found, then ...". I definitely prefer "if is not found, then". Of course you need to move the code blocks below into proper place - I couldn't force GitHub to make the suggestion span more lines.
```suggestion
		if util.IsNotFound(err) {
```